.agents/skills/auto-verify/SKILL.md

```
@@ -0,0 +1,73 @@
+- Use when a TaskBundle expects verification evidence or when you are preparing an implementation receipt.
+- Do not skip this skill just because the change looks small.
+
+## Pipeline
+
+1. Collect the changed files:
+
+```powershell
+git diff --cached --name-only
+```
+
+2. Run type-check or compile checks when the project supports them. Prefer the repo's existing package scripts; otherwise use a direct compiler invocation.
```

markdownlint-cli2 (0.21.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)